policy/modules/system/udev.if

```diff
@@ -202,6 +202,50 @@ interface(`udev_manage_rules_files',`
 	udev_search_runtime($1)
 ')
 
+########################################
+## <summary>
+##	Relabel udev rules directories
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`udev_relabel_rules_dirs',`
+	gen_require(`
+		type udev_rules_t;
+	')
+
+	relabel_dirs_pattern($1, udev_rules_t, udev_rules_t)
+
+	files_search_etc($1)
+
+	udev_search_runtime($1)
```

Comment:
how is `/run/udev` related to this?

Reply:
Looking over it again, I don't think it is. That access was copied from the above interfaces when I made these new ones. Notwithstanding the above (whether these interfaces are needed at all), this access can be removed.